app/_locales/en/messages.json

```diff
@@ -2597,6 +2597,9 @@
   "slow": {
     "message": "Slow"
   },
+  "snapAccess": {
+    "message": "$1 snap has access to:"
```

Comment:
Whenever we use a wildcard like `$1`, we add a `description` property to the object which explains what the variable represents.

In this case, I'd make it:

```
"snapAccess": {
  "message": "$1 snap has access to:",
  "description":  "$1 represents the name of the snap"
},
```